spec/factories/transfer_request_collections.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :transfer_request_collection do
+    transient do
+      target_tube_count { 1 }
+      target_assets do
+        Array.new(target_tube_count) { create(:receptacle, labware: create(:stock_multiplexed_library_tube)) }
```

Comment:
Do the target assets need to be receptacles? Can they be labware directly?

As a sidenote I think another way of writing this would be:
`target_assets { create_list(:receptacle, target_tube_count, labware: create(:stock_multiplexed_library_tube) }`

Reply:
Taken from the TransferRequest model: `# target_asset and asset are both Receptacle objects, and are the source and target of the transfer request.`.  I used a real TransferRequestCollection to determine how to put one of these together.

I will use your alternative way of defining the list though.  I did know about this method, but I originally created a set of tubes in a separate transient and plucked them one at a time from that list.  Until I moved the `create` inline here and then I should have gone back to `create_list`.

Reply:
Cool thanks for the explanation.